### ISSUE TITLE
Add label to hidden input for accessibility

### ISSUE THREAD

**Issue by SRTM** [lcdsantos/jQuery-Selectric#195]
When I run my accessibility checker, Selectric fails because it is missing a <label> for the hidden input Selectric creates. Is there a way you can add a hidden <label> as well to accompany the hidden input?
See Image attached.

![selectric-label-accessibility](https://user-images.githubusercontent.com/4974044/28436016-7c15e4a2-6d63-11e7-9521-15f1f9b90622.jpg)

Thanks

**Comment by SRTM** [comment]
I've coded a temporary fix for this:
''//Add a hidden label around the selectric input.
	$(".selectric-input").wrap("<label class='VisuallyHidden' aria-hidden='true'>Hidden Label</label>");''

Where VisuallyHidden is similar to the ''selectric-input'' class in hiding the label.

**Comment by hh-com** [comment]
Works for me...
`
....
onInit: function(element)
{
      $(element).closest('.selectric-wrapper').find('.selectric-input').attr('id', element[0].text);
},
....
`

**Comment by zunz** [comment]
Modified from @hh-com 
`$('select').selectric({
onInit: function(element) { $(element).closest('.selectric-wrapper').find('.selectric-input').attr('type', 'hidden'); }
	});`
